src/web/app/pages-instructor/instructor-course-student-details-page/instructor-course-student-details-page.component.html

```diff
@@ -1,3 +1,128 @@
-<p>
-  instructor-course-student-details-page works!
-</p>
+<ng-template #notSpecified><i class="text-muted">Not Specified</i></ng-template>
+<div *ngIf="student">
+  <h1>{{student.name}}</h1>
+  <div class="row" *ngIf="studentProfile">
+    <div class="col-12">
+      <div class="row">
+        <div class="col-md-2 col-3">
+          <img src="{{getPictureUrl(studentProfile.pictureKey)}}" class="profile-pic float-right">
+        </div>
+        <div class="col-md-10 col-sm-9 col-8">
+          <table class="table table-striped">
+            <thead>
+              <tr>
+                <th colspan="2"><h4><span class="font-weight-bold">Profile Details </span>(submitted by student)</h4></th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td class="font-weight-bold">Short Name (Gender)</td>
+                <td><span *ngIf="studentProfile.shortName; else notSpecified">{{studentProfile.shortName}}</span>
+                  (<i *ngIf="studentProfile.gender != 'other'; else notSpecified">{{studentProfile.gender}}</i>)
+                </td>
+              </tr>
+              <tr>
+                <td class="font-weight-bold">Personal Email</td>
+                <td><span *ngIf="studentProfile.email; else notSpecified">{{studentProfile.email}}</span>
```

Comment:
hii dat i m referring to ur code and realized u didn't close the `<td>`s...

Reply:
My bad. I think you can just add it in your pr?